service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapper.java

```diff
@@ -85,6 +92,7 @@ private String mapListResourceToComponents(ListResource listReferencedToEncounte
 
     private Optional<String> mapItemToComponent(ListResource.ListEntryComponent item) {
         Optional<Resource> resource = messageContext.getInputBundleHolder().getResource(item.getItem().getReferenceElement());
+        resource.map(Resource::getResourceType).ifPresent(System.out::println);
 
         return resource.map(value -> encounterComponents.getOrDefault(value.getResourceType(), this::mapDefaultNotImplemented)
```

Comment:
As above about phasing out our placeholder

```suggestion
        return resource.map(value -> encounterComponents.get(value.getResourceType()).apply(value));
```

Reply:
Nevermind, I'll do this in the UAT ticket